bin/versions/fetchVersions.mjs

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch";
+import { writeFileSync } from "fs";
+import path from "path";
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+const FILE_PATH = path.resolve(__dirname, "../data/pages/releases/versions.json");
+
+(async () => {
+  try {
+    const response = await fetch("https://openui5versiontracker.cfapps.eu10.hana.ondemand.com/OpenUI5ReleasesInfo");
```

Comment:
is it possible to move to the variable?

Reply:
move what?

Reply:
I meant the url. Is it possible as in the script?

Reply:
done